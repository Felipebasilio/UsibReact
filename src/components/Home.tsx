import React from 'react';
import * as torno3 from "../assets/torno3.jpeg";
import * as torno1 from "../assets/torno1.jpeg";
import * as tornop from "../assets/tornopolietileno.jpeg";
import * as tornoFresa from "../assets/tornofresa.jpeg";
import * as tornoCNC from "../assets/tornocnc.jpeg";

const radiousDivStyle = { 
  width: "80%",
  height: "300px",
  borderRadius: "21px 21px 0 0"
}
const Home = () => {
  return (
    <>
      <main className="vh-100">
        <div className="position-relative overflow-hidden p-3 p-md-5 m-md-3 text-center bg-light">
          <div className="col-md-5 p-lg-5 mx-auto my-5">
            <h1 className="display-4 fw-normal">USIB</h1>
            <p className="lead fw-normal">A solução perfeita para os casos de usinagem,   fresa e tornearia.</p>
          </div>
          <div className="shadow-sm d-none d-md-block">
            <img alt="Torno 3" src={torno3.default}/>
          </div>
        </div>
      
        <div className="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3">
          <div className="bg-dark me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center   text-white overflow-hidden">
            <div className="my-3 py-3">
              <h2 className="display-5">Aço</h2>
            </div>
            <img className="bg-light shadow-sm mx-auto" style={ radiousDivStyle } alt="Torno 1" src={torno1.default}/>
          </div>
          <div className="bg-light me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center  overflow-hidden">
            <div className="my-3 p-3">
              <h2 className="display-5">Plástico</h2>
            </div>
            <img className="bg-light shadow-sm mx-auto" style={ radiousDivStyle } alt="Torno Polietileno" src={tornop.default}/>
            
          </div>
        </div>
        
        <div className="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3">
          <div className="bg-light me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center  overflow-hidden">
            <div className="my-3 p-3">
              <h2 className="display-5">Fresa</h2>
            </div>
            <img className="bg-light shadow-sm mx-auto" style={ radiousDivStyle } alt="Torno fresa" src={tornoFresa.default}/>
          </div>
          <div className="bg-dark me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center   text-white overflow-hidden">
            <div className="my-3 py-3">
              <h2 className="display-5">CNC</h2>
            </div>
            <img className="bg-light shadow-sm mx-auto" style={ radiousDivStyle } alt="Torno CNC" src={tornoCNC.default}/>
          </div>
        </div>
      
      
      </main>
    </>


  );
};

export default Home;