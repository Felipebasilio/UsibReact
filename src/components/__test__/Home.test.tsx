import ReactDOM from 'react-dom';
import { cleanup } from '@testing-library/react';
import Home from '../Home';

import renderer from 'react-test-renderer'

afterEach(cleanup);

it('renders Home without crashing', () => {
    let div = document.createElement('div');
    ReactDOM.render( <Home />, div );
    ReactDOM.unmountComponentAtNode(div);
});


it('Matches snapshot', () => {
    const HomeComponnet = renderer.create(<Home />).toJSON();
    expect(HomeComponnet).toMatchSnapshot();
});
