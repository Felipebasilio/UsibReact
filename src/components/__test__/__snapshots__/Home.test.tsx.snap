// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<main
  className="vh-100"
>
  <div
    className="position-relative overflow-hidden p-3 p-md-5 m-md-3 text-center bg-light"
  >
    <div
      className="col-md-5 p-lg-5 mx-auto my-5"
    >
      <h1
        className="display-4 fw-normal"
      >
        USIB
      </h1>
      <p
        className="lead fw-normal"
      >
        A solução perfeita para os casos de usinagem,   fresa e tornearia.
      </p>
    </div>
    <div
      className="shadow-sm d-none d-md-block"
    >
      <img
        alt="Torno 3"
        src="torno3.jpeg"
      />
    </div>
  </div>
  <div
    className="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3"
  >
    <div
      className="bg-dark me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center   text-white overflow-hidden"
    >
      <div
        className="my-3 py-3"
      >
        <h2
          className="display-5"
        >
          Aço
        </h2>
      </div>
      <img
        alt="Torno 1"
        className="bg-light shadow-sm mx-auto"
        src="torno1.jpeg"
        style={
          Object {
            "borderRadius": "21px 21px 0 0",
            "height": "300px",
            "width": "80%",
          }
        }
      />
    </div>
    <div
      className="bg-light me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center  overflow-hidden"
    >
      <div
        className="my-3 p-3"
      >
        <h2
          className="display-5"
        >
          Plástico
        </h2>
      </div>
      <img
        alt="Torno Polietileno"
        className="bg-light shadow-sm mx-auto"
        src="tornopolietileno.jpeg"
        style={
          Object {
            "borderRadius": "21px 21px 0 0",
            "height": "300px",
            "width": "80%",
          }
        }
      />
    </div>
  </div>
  <div
    className="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3"
  >
    <div
      className="bg-light me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center  overflow-hidden"
    >
      <div
        className="my-3 p-3"
      >
        <h2
          className="display-5"
        >
          Fresa
        </h2>
      </div>
      <img
        alt="Torno fresa"
        className="bg-light shadow-sm mx-auto"
        src="tornofresa.jpeg"
        style={
          Object {
            "borderRadius": "21px 21px 0 0",
            "height": "300px",
            "width": "80%",
          }
        }
      />
    </div>
    <div
      className="bg-dark me-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center   text-white overflow-hidden"
    >
      <div
        className="my-3 py-3"
      >
        <h2
          className="display-5"
        >
          CNC
        </h2>
      </div>
      <img
        alt="Torno CNC"
        className="bg-light shadow-sm mx-auto"
        src="tornocnc.jpeg"
        style={
          Object {
            "borderRadius": "21px 21px 0 0",
            "height": "300px",
            "width": "80%",
          }
        }
      />
    </div>
  </div>
</main>
`;
